tests/models/blip_2/test_modeling_blip_2.py

```diff
@@ -704,6 +716,16 @@ class Blip2ModelTest(ModelTesterMixin, PipelineTesterMixin, GenerationTesterMixi
     test_attention_outputs = False
     test_torchscript = False
 
+    # TODO: Fix the failed tests
```

Comment:
To be addressed?

Reply:
Not on this PR, I don't believe it is related to the changes of this PR, Blip2ForConditionalGeneration fails there, but i will verify that.

I wanted to make the test pass, and leave a comment about this, as i saw similar comments on other models.

Reply:
this is the error i was getting, it also occurs on the main branch

`FAILED tests/models/blip_2/test_modeling_blip_2.py::Blip2ModelTest::test_pipeline_visual_question_answering_fp16 - RuntimeError: "LayerNormKernelImpl" not implemented for 'Half'`

Reply:
@jpizarrom Could you open an issue to track this to make sure this isn't lost?

Reply:
Refer to [this comment](https://github.com/huggingface/transformers/issues/21989#issuecomment-1461999591) on why the test fails. You're probably running this on CPU.